test/Microsoft.ML.Functional.Tests/ModelFiles.cs

```diff
@@ -217,6 +217,7 @@ public class ModelInput
             public string[] CategoricalFeatures;
             public float[] NumericalFeatures;
 #pragma warning restore SA1401
+            public float Label;
```

Comment:
>Label [](start = 25, length = 5)

add extra column Label as the model modelwithoptionalcolumntransform's schema contains an extra Label column